src/beacon.js

```diff
@@ -152,4 +152,9 @@ export default function Emitter(endpoint, context, options) {
   };
 
   this.flush = transmit;
+
+  // This exists to clear messages in tests. There is no reason to call it in normal usage.
+  this.clearMessages = function() {
```

Comment:
Can it be named like `_clearMessages` or `_clearMessagesForTesting` or somesuch?